src/ddestats.jl

```diff
@@ -0,0 +1,34 @@
+mutable struct DDEStats
+  nf::Int
+  nf2::Int
+  nw::Int
+  nsolve::Int
+  njacs::Int
+  nnonliniter::Int
+  nnonlinconvfail::Int
+  nfpiter::Int
+  nfpconvfail::Int
+  ncondition::Int
+  naccept::Int
+  nreject::Int
+  maxeig::Float64
+end
+
+DDEStats(x::Int = -1) = DDEStats(x, x, x, x, x, x, x, x, x, x, x, x, 0.0)
+
+function Base.show(io::IO, s::DDEStats)
+  println(io, summary(s))
+  @printf io "%-60s %-d\n" "Number of function 1 evaluations:" s.nf
```

Comment:
Should the numbers be left or right aligned?

Reply:
I don't have an opinion on that.

Reply:
Should DiffEqBase use `@printf` as well so the numbers are aligned?

Reply:
Yeah it probably should. That would make it much nicer.